google/cloud/logging_v2/handlers/app_engine.py

```diff
@@ -114,11 +114,14 @@ def emit(self, record):
         """
         message = super(AppEngineHandler, self).format(record)
         gae_labels = self.get_gae_labels()
-        trace_id = (
-            "projects/%s/traces/%s" % (self.project_id, gae_labels[_TRACE_ID_LABEL])
-            if _TRACE_ID_LABEL in gae_labels
-            else None
-        )
+        http_request, trace_id = get_request_data()
+        if trace_id is not None:
+            trace_id = f"projects/{self.project_id}/{trace_id}"
```

Comment:
Is it `f"projects/{self.project_id}/{trace_id}"`
or is it f"projects/{self.project_id}/traces/{trace_id}"

Reply:
Good catch! I'm not too familiar with tracing yet so I don't know what it *should* be, but I meant to copy what was being done previously. I'll have to add a test to check for this